nextflow/workspace_description.md

```diff
@@ -0,0 +1,48 @@
+# Overview
+
+This notebook shows how to get started with [Nextflow](https://www.nextflow.io) on Enterprise Terra.
+
+# How do I get started?
+
+## Step 0: _Duplicate_ this workspace
+
+- *Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace. Duplicating this workspace ensures three Git repositories required for these examples will be automatically cloned when you create a Cloud Environment.
+
+   The required repositories which will be cloned include:
+  1. [`terra-axon-examples` example repo](https://github.com/DataBiosphere/terra-axon-examples): Contains JupyterLab notebook `nextflow_examples.ipynb` which demonstrates how to configure and run Nextflow workflows in Enterprise Terra.
+  2. [`rnaseq-nf` repo](https://github.com/nextflow-io/rnaseq-nf): Contains a Nextflow RNASeq workflow specification and associated human genomic input data.
+  3. [`test-datasets` repo](https://github.com/nf-core/test-datasets): For each workflow in the `nf-core` collection, this repo has a branch with appropriate, workflow-specific test data.
+  > **Note**: If you do not have a GitHub account, then you can clone each repo manually after you've created the Cloud Environment.
+
+
+## Step 1: Preview prior runs of the relevant notebooks.
+
+To gain a better understanding of the setup and analysis included in this demonstration workspace, previews of this notebook that include cell outputs are provided. Navigate to your workspace's Resources tab, then select the "Notebook snapshots" folder. To view a notebook snapshot, select a file in said folder and click the "Preview" button.
+
+## Step 2: Create an Enterprise Terra Cloud Environment and run a setup notebook.
+
+Create an Enterprise Terra Cloud environment, by navigating to the "Environments" tab of the workspace. You can use the configuration defaults.
+
+Launch the environment once it's running, and then run the notebook [workspace_setup.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/workspace_setup.ipynb). Its repo, `terra-axon-examples`, which is defined as a workspace Git repository, should be automatically cloned to your Enterprise Terra Cloud Environment, and you should be able to navigate to the notebook in the JupyterLab file browser. Look for the `terra-axon-examples` subdirectory.
+
+> **If you do not see the `terra-axon-examples` subdirectory**, open a Terminal window on the notebook server (under the **File** menu) and run:
```

Comment:
Something changed slightly here in the last commit, because this now looks like 
![image](https://github.com/DataBiosphere/terra-axon-examples/assets/6879774/df9b8a46-379f-467e-a401-e6cdfb07bfa9)

Reply:
That's strange. This is how the content renders in the Jupyter Markdown preview of the file: https://screenshot.googleplex.com/9Ycw9AsLNkVMxD4.

I'll look into whether this is a nuance of GitHub Markdown.

Reply:
Fixed.